test/utils/azurefile_log.sh

```diff
@@ -38,6 +38,13 @@ kubectl get pods -n${NS} -l${LABEL} \
     | awk 'NR>1 {print $1}' \
     | xargs -I {} bash -c "echo 'dumping logs for ${NS}/{}/${CONTAINER}' && kubectl logs {} -c${CONTAINER} -n${NS}"
 
+#echo "print out csi-snapshot-controller pods ..."
```

Comment:
uncomment those lines?

Reply:
that's ok, need to uncomment when sometimes snapshot function is broken